src/Gui/Forms/InitialPageInteractor.cs

```diff
@@ -182,5 +192,78 @@ private bool NeedsScanning()
                 Decompiler.Project != null &&
                 Decompiler.Project.Programs.Any(p => p.NeedsScanning);
         }
+
+        private void ProcessResources(ProgramResourceGroup prg, string outputDir)
+        {
+            try
+            {
+                if (prg.Name == "PE resources")
+                {
+                    // Create the dir
+                    Directory.CreateDirectory(outputDir + "\\resources");
```

Comment:
Platform-specific code

Reply:
Consider using `Path.Combine` to build the path name. 

In Reko, to make program code unit testable, we don't write code directly to `System.IO.Directory` or  `System.IO.File`. Instead, we call methods on the interface `IFileSystemService`. This way we can simulate file systems in unit tests. At the top of this method, you should add the line:
```C#
var fsSvc = Services.RequireService<IFileSystemService>();
```
and then replace this call with:
```C#
fsSvc.CreateDirectory(Path.Combine(outputDir, "resources");
```
Since that method doesn't exists on `Reko.Core.Services.IFileSystemService` yet, please add it to the interface and the class `Reko.Core.Services.FileSystemImpl`.

It is possible the user may want to rename the `resources` directory, but we can look at that later.